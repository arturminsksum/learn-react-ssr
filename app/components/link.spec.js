import React from 'react';
import Link from './link';

describe('Link', () => {
  it('should render clickable link', () => {
    const wrapper = shallow(<Link />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render non-clickable link', () => {
    const wrapper = shallow(<Link active={true} />);
    expect(wrapper).toMatchSnapshot();
  });
});
