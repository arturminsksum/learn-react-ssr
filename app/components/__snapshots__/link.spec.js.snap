// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render clickable link 1`] = `
<li>
  <a
    href=""
    onClick={[Function]}
  />
</li>
`;

exports[`Link should render non-clickable link 1`] = `
<li
  className="is-active"
>
  <a
    href="javascript:void(0)"
  />
</li>
`;
